api/src/main/java/org/openmrs/module/eptsreports/reporting/utils/EptsCommonUtils.java

```diff
@@ -0,0 +1,115 @@
+package org.openmrs.module.eptsreports.reporting.utils;
+
+import java.util.Date;
+import java.util.List;
+import org.openmrs.EncounterType;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.Program;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.eptsreports.metadata.HivMetadata;
+import org.openmrs.module.eptsreports.reporting.calculation.data.quality.PatientDemographicsCalculation;
+import org.openmrs.module.eptsreports.reporting.cohort.definition.CalculationDataDefinition;
+import org.openmrs.module.eptsreports.reporting.library.converter.CalculationResultDataConverter;
+import org.openmrs.module.reporting.common.Birthdate;
+import org.openmrs.module.reporting.common.TimeQualifier;
+import org.openmrs.module.reporting.data.DataDefinition;
+import org.openmrs.module.reporting.data.converter.BirthdateConverter;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.reporting.data.converter.ObjectFormatter;
+import org.openmrs.module.reporting.data.converter.PropertyConverter;
+import org.openmrs.module.reporting.data.patient.definition.ConvertedPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.EncountersForPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PatientIdentifierDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.PersonToPatientDataDefinition;
+import org.openmrs.module.reporting.data.patient.definition.ProgramEnrollmentsForPatientDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.GenderDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PersonDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PreferredNameDataDefinition;
+import org.openmrs.module.reporting.dataset.definition.PatientDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EptsCommonUtils {
+
+  @Autowired private HivMetadata hivMetadata;
+  /**
+   * Adds the standard patient list columns
+   *
+   * @param dsd the data set definition
+   */
+  public void addStandardColumns(PatientDataSetDefinition dsd) {
```

Comment:
This is not really a utility function. Better that we extend EptsReportManager for datasets and place it there. Similar to where we placed the other `addColumn` functions.